action.yaml

```
@@ -17,15 +17,23 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Download Coder binary from access URL
-      run: curl -fsSL $CODER_URL/bin/coder-linux-amd64 -o /usr/local/bin/coder && chmod +x /usr/local/bin/coder
+    - name: Download and install Coder binary
+      run: |
+        if [ "${{ runner.arch }}" == "ARM64" ]; then
+          ARCH="arm64"
+        else
+          ARCH="amd64"
+        fi
+        mkdir -p $HOME/.local/bin
```

Good practice to quote all variables, i.e.
```suggestion
        mkdir -p "$HOME"/.local/bin
```